example_bots/server.py

```diff
@@ -223,3 +225,17 @@ def pong():
         return {'author': 'PongBot', 'text': 'Please activate PongBot via the PingBot.'}
 
     return {'author': 'PongBot', 'text': 'This is PongBot. How are you today?', 'state': 0}
+
+
+@app.post('/buttonbot')
+def buttonbot():
+    """ButtonBot is an example of how to use resource-linked CSS and Javascript."""
+    return {
+        'author': 'ButtonBot',
+        'text': '''
+            <p>Click this button to run good Javascript: <button class="buttonbot" onclick="buttonBotClick()">Click me!</button></p>
+            <p>Click this button to throw a JS error: <button class="buttonbot-error" onclick="buttonBotError()">Don't click me!</button></p>
+        ''',
+        'css': '/static/buttonbot.css',
```

Comment:
should this be `./static/buttonbot.css` ? or is absolute path fine

Reply:
The logic is just `urllib.parse.urljoin(bot_url, script_url)`.

```python
>>> from urllib.parse import urljoin
>>> urljoin('https://joels.best.bot/mybot', 'static/script.js')
'https://joels.best.bot/static/script.js'
>>> urljoin('https://joels.best.bot/mybot', 'static/script.js')
'https://joels.best.bot/static/script.js'
>>> urljoin('https://joels.best.bot/mybot', '/static/script.js')
'https://joels.best.bot/static/script.js'
>>> urljoin('https://joels.best.bot/mybot', 'https://some.other.site/static/script.js')
'https://some.other.site/static/script.js'
```